ArcGISRuntimeSDKQt_QMLSamples/Layers/ListKmlContents/ListKmlContents.qml

```diff
@@ -231,6 +330,111 @@ Rectangle {
                 ArcGISTiledElevationSource {
                     url: "http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"
                 }
+
+                onLocationToElevationStatusChanged: {
+                    if (locationToElevationStatus !== Enums.TaskStatusCompleted)
+                        return;
+
+                    // assume altitude mode is clamp-to-ground if not specified
+                    let altMode = Enums.KmlAltitudeModeClampToGround;
+                    let kmlViewpoint = currentNode.viewpoint;
+
+                    if (kmlViewpoint !== undefined && kmlViewpoint !== null) {
+                        altMode = kmlViewpoint.altitudeMode;
+                    }
```

Comment:
there's a bit of a mixture of styles with regards brackets around single line if statements. i'm not sure what our code style for samples is - but good to be consistent within the one file.